index.html

```diff
@@ -0,0 +1,117 @@
+            </a>
+          </li>
+          <li>
+            <a href="">
+              <img src="images/twit-icon.png" alt="twitter">
+            </a>
+          </li>
+          <li>
+            <a href="">
+              <img src="images/gp-icon.png" alt="google plus">
+            </a>
+          </li>
+          <li>
+            <a href="">
+              <img src="images/insta-icon.png" alt="instagram">
+            </a>
+          </li>
+          <li>
+            <a href="">
+              <img src="images/flic-icon.png" alt="flickr">
+            </a>
+          </li>
+          <li>
+            <a href="">
+              <img src="images/pint-icon.png" alt="pintrest">
+            </a>
+          </li>
+          <li>
+            <a href="">
+              <img src="images/rss-icon.png" alt="rss">
+            </a>
+          </li>
+          <li>
+            <a href="">
+              <img src="images/mail-icon.png" alt="mail">
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <div class="border"></div>
+    <main>
+      <article>
+        <h2 class="recipeHeader">Chocolate Pizza</h2>
+        <h3 class="postDate">POSTED ON 15 DEC 2013<time datetime="2013-12-13" pubdat>/DESSERTS</h3>
+        <button class="printButton">
+          <img src="images/print-icon.png"> PRINT
+        </button>
+        <img src="images/choco-pizza.png" alt="chocolate pizza" class="pizzaPhoto">
+        <p>For the fig-swirl: Melt butter over medium heat in a saucepan. Add brown sugar and stir to dissolve. Halve all of the figs and toss in a saucepan with water and lemon juice. Cook over medium heat, stirring frequently, until you have a chunky-jammy mixature. Add salt with one or two stirs, set aside and let cool completely.</p>
+        <p>Ice cream: in a small pot over medium heat, combine milk, and granulated sugar until sugar is completely dissolved and the milk is just barely lukewarm. Whisk in the egg yolks. Set mixture in the fridge and wait until the fig mixture is cooled.</p>
+        <p>Using an ice cream machine, pour liquids into the frozen basin and process according to manufacturer instructions, i.e., let spin and thicken for 20 minutes. Pour semi-frozen mixture into a pyrex dish or glass tupperware. Freeze for at least two hours before serving.</p>
+        <section class="ingredients">
+          <ul>
+            <li><input type="checkbox">1 1/2 cups milk</li>
+            <li><input type="checkbox">1/2 cup mascarpone</li>
+            <li><input type="checkbox">1/2 tsp pink salt</li>
+            <li><input type="checkbox">1lb Black Mission Figs</li>
+            <li><input type="checkbox">1/2 cup brown sugar</li>
+            <li><input type="checkbox">2-4 tbsp water</li>
+          </ul>
+          <ul>
+            <li><input type="checkbox">1 1/2 cups heavy cream</li>
+            <li input type = "checkbox">1/3 granulated sugar</li>
```

Comment:
92-93: You opening `li` tags are each missing a `>` bracket. Remove the spacing between the `=` sign between the type and "checkbox".